config/metadata/child_works_from_pdf_splitting.yaml

```diff
@@ -4,13 +4,19 @@ attributes:
     multiple: false
     index_keys:
       - "is_child_bsi"
+    form:
```

Comment:
I wonder if adding this back will set is_child to false again, instead of nil. cc @laritakr

Reply:
Also ideally we won't update the query because existing applications would need to set their nils to false. Unless the query can filter out both 🤔

Reply:
perhaps something like (untested)

```ruby
def show_parents_only(solr_parameters)
  query = if blacklight_params["include_child_works"] == 'true'
            IiifPrint.solr_construct_query(is_child_bsi: 'true')
          else
            "-is_child_bsi:true"
          end
  solr_parameters[:fq] += [query]
end
```

Reply:
The term is not present on the parent, so it is nil.
On the child, it is true as expected. 
The child works are once again hidden from the catalog search.

![Screenshot 2024-06-20 at 3 14 17 PM](https://github.com/scientist-softserv/iiif_print/assets/17851674/01e42285-9a09-4426-915d-42ffa8
![Screenshot 2024-06-20 at 3 14 45 PM](https://github.com/scientist-softserv/iiif_print/assets/17851674/07b6a3c2-d56e-471d-84cd-e78770c5d4a2)
6a9
![Screenshot 2024-06-20 at 3 15 14 PM](https://github.com/scientist-softserv/iiif_print/assets/17851674/a7253f7c-5aaf-40c4-bd86-8550ecd18a22)
be6)